File: 1200/2196A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            long p = sc.nextLong();
            long q = sc.nextLong();
            
            if(3*p < 2*q || p >= q)
                System.out.println("Alice");
            else
                System.out.println("Bob");
        }
    }
}

File: 1200/2196A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        long long p, q;
        cin >> p >> q;

        if (3 * p < 2 * q || p >= q)
            cout << "Alice\n";
        else
            cout << "Bob\n";
    }

    return 0;
}
